Gemfile

```diff
@@ -0,0 +1,42 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.0.2'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails', '~> 6.1.7', '>= 6.1.7.8'
```

Comment:
why not use newer versions?

Reply:
Just because those were the ones i had on my local machine on the previous company